### PR TITLE
docs: Add select bindParam example

### PR DESCRIPTION
**Pull request by jhonnattan123**

### CODE REVIEW

**Review by abmusse — CHANGES_REQUESTED** [review]
The changes overall look good.

Before this can be merged though need to pass the [DCO check](https://github.com/IBM/nodejs-idb-pconnector/pull/101/checks?check_run_id=1221730828)

Also we use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#examples) for commit messages. Consider updating the commit message to something like: 

`docs: Add select prepare, bind, execute example`

**Review by jhonnattan123 — COMMENTED** [review]
docs: Add select bindParam example

**Comment by jhonnattan123** [comment]
> The changes overall look good.
> 
> Before this can be merged though need to pass the [DCO check](https://github.com/IBM/nodejs-idb-pconnector/pull/101/checks?check_run_id=1221730828)
> 
> Also we use [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/#examples) for commit messages. Consider updating the commit message to something like:
> 
> `docs: Add select bindParam example`

is ready

File: .github/workflows/stale.yml [hunk 1]
```diff
@@ -16,9 +16,9 @@ jobs:
         stale-issue-message: ':wave: Hi! This issue has been marked stale due to inactivity. If no further activity occurs, it will automatically be closed.'
         stale-pr-message: ':wave: Hi! This pull request has been marked stale due to inactivity. If no further activity occurs, it will automatically be closed.'
         stale-issue-label: 'stale'
-        exempt-issue-label: 'keep-open'
-        exempt-pr-label: 'keep-open'
         stale-pr-label: 'stale'
+        exempt-issue-labels: 'keep-open'
+        exempt-pr-labels: 'keep-open'
```
Comment (abmusse) [inline .github/workflows/stale.yml:21]: image

Hey @jhonnattan123 Look `fix: Make exempt labels plural for stale action (#100)` was committed with this PR.  This changes is already merged in the master branch. Could you drop that commit from your branch. Thanks!

**Review by abmusse — APPROVED** [review]
LGTM

**Pull request merged**